vault/changelog.release.2022-08-09.md

```diff
@@ -0,0 +1,71 @@
+---
+id: 482bl67lvjd0r8ht0cngxmk
+title: 0.107
+desc: ''
+updated: 1659972909881
+created: 1659968708044
+---
+
+Dendron 0.107 has sprouted  🌱
+August 09, 2022
+
+**Add Toggle Preview Lock command**: You can now lock in the preview of a specific note and navigate other notes without changing your preview.
+
+- More information: [[Toggle Preview Lock|dendron://dendron.dendron-site/dendron.topic.preview.commands.toggle-preview-lock]]
+
+<div style="position: relative; padding-bottom: 64.5933014354067%; height: 0;"><iframe src="https://www.loom.com/embed/6e35934a0c534f3585e750e5eef2e739" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>
+
+## Highlights
+
+- feat(view): add "Toggle PreviewLock" command (#3293) @sam ([[docs|dendron.topic.preview.commands.toggle-preview-lock]])
+
+## Everything Else
+
+- enhance(structure): support xvault template in note traits (#3329) @joshi ([[docs|dendron://dendron.dendron-site/dendron.topic.traits.api#return]])
```

Comment:
do not include `@{user}` or pr links `(#....)` in the release note

Reply:
how about the docs link?

Reply:
does the existing link not work?